exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -232,15 +232,27 @@ def handle_redirect(location)
           # TODO: figure out destination and reinitialize @http and @path
         end
 
-        def log_status(body)
-          status = Google::Rpc::Status.decode(body)
-          details = status.details.map do |detail|
-            klass_or_nil = ::Google::Protobuf::DescriptorPool.generated_pool.lookup(detail.type_name).msgclass
-            detail.unpack(klass_or_nil) if klass_or_nil
-          end.compact
-          OpenTelemetry.handle_error(message: "OTLP exporter received rpc.Status{message=#{status.message}, details=#{details}}")
-        rescue StandardError => e
-          OpenTelemetry.handle_error(exception: e, message: 'unexpected error decoding rpc.Status in OTLP::Exporter#log_status')
+        def log_status(response) # rubocop:disable Metrics/AbcSize
+          content_type = response['Content-Type']
+          code = response.code
+          body = response.body
+
+          http_status_message = "http.code=#{code} http.content_type='#{content_type}'"
+          case content_type
+          when 'text/plain; charset=utf-8'
```

Comment:
I am a little torn here. Should this code explicitly check for the protobuf content type instead and treat anything else as an error?

Reply:
Yes, I've been thinking about this some more and I think it would be better to take a slightly different approach. This change is meant to be about logging 404 errors cleanly so really it should checking for that code and creating a log message that is 404 specific.